staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -275,7 +275,7 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 func (e *eventBroadcasterImpl) StartLogging(logf func(format string, args ...interface{})) watch.Interface {
 	return e.StartEventWatcher(
 		func(e *v1.Event) {
-			logf("Event(%#v): type: '%v' reason: '%v' %v", e.InvolvedObject, e.Type, e.Reason, e.Message)
```

Comment:
This change is wrong. You need to change the callers of this function, not the implementation here.

Reply:
Hey Daniel, can you take a look at my proposal to change the interface?